__tests__/app.test.js

```diff
@@ -360,3 +360,24 @@ describe("Comments Endpoint", () => {
     });
   });
 });
+
+describe("Users Endpoint", () => {
+  describe("GET:/api/users", () => {
+    test("GET:200 - Responds with an array containing correctly formated user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          const users = body.users;
+          expect(users.length).not.toBe(0);
```

Comment:
You should be more specific here, at least using `toBeGreaterThan` or even `toBe` and use the exact number